### DISCUSSION TITLE
Jyers support for V6 Volcano Hotends

### DISCUSSION THREAD

**Discussion by thanatas** [Jyers/Marlin#1381] [Ideas]
Is it possible to generate release for V6 volcano hotends?  it only request to change thermistor's parameter. It just needs a small config difference. Doing it by source code is very hard without ci/cd.

I appreciated all works here, We are so lucky...

Thanks.

**Comment by Cal-LF** [comment]
A version to support only one type of hot end would not be practical. You can change the settings by compiling your own. There is detailed instructions on utube videos and here in Jyers issues and discussions areas. If you want to learn to compile Jyers FW, I can help get you going. It is not difficult to use VS Code to edit the configuration text files.

**Comment by JCRodzDom** [comment]
Guys I tried to compile the firmware to a higher temp but the out put was an .hex not a .bin 
And the printer will not take it.
